encodings/dict/src/array.rs

```diff
@@ -83,10 +87,14 @@ impl CanonicalVTable<DictArray> for DictEncoding {
 
 impl ValidityVTable<DictArray> for DictEncoding {
     fn is_valid(&self, array: &DictArray, index: usize) -> VortexResult<bool> {
-        let values_index = scalar_at(array.codes(), index)
-            .unwrap_or_else(|err| {
-                vortex_panic!(err, "Failed to get index {} from DictArray codes", index)
-            })
+        let scalar = scalar_at(array.codes(), index).unwrap_or_else(|err| {
+            vortex_panic!(err, "Failed to get index {} from DictArray codes", index)
```

Comment:
I don't really understand why we're panicking here?

Reply:
just C&P, but I have removed it